sanic/server.py

```diff
@@ -42,6 +45,40 @@ class Signal:
     stopped = False
 
 
+class ConnInfo:
+    """Local and remote addresses and SSL status info."""
+
+    __slots__ = (
+        "sockname",
+        "peername",
+        "server",
+        "server_port",
+        "client",
+        "client_port",
+        "ssl",
+    )
+
+    def __init__(self, transport):
+        self.ssl = bool(transport.get_extra_info("sslcontext"))
+        self.sockname = a = transport.get_extra_info("sockname")
```

Comment:
It's a strictly local variable, a very short name for one in purpose. Otherwise one could just use `self.sockname` and `self.peername` but it makes the code unnecessarily long.

Alike, multiple assignment should be pretty standard, right?

Reply:
Yes, I understand your point, but in my opinion, it would be better to rename `a` to something a bit more understandable, also don't forget about Python's zen:)
```
Explicit is better than implicit.
```

Naming is a very important thing and on another hand, it always hard to choose right naming:)

Reply:
I think this is explicit enough. As said, it is not exposed outside. I don't have a strong opinion.